diff --git a/include/external/local_tsl/tsl/platform/ctstring.h b/include/external/local_tsl/tsl/platform/ctstring.h
index f841e5f..bcfe8ae 100755
--- a/include/external/local_tsl/tsl/platform/ctstring.h
+++ b/include/external/local_tsl/tsl/platform/ctstring.h
@@ -19,7 +19,7 @@ limitations under the License.
 #include <stdint.h>
 #include <stdlib.h>
 
-#include "tsl/platform/ctstring_internal.h"
+#include "external/local_tsl/tsl/platform/ctstring_internal.h"
 
 // Initialize a new tstring.  This must be called before using any function
 // below.
diff --git a/include/tensorflow/c/tf_status.h b/include/tensorflow/c/tf_status.h
index 31e5a4f..bf1c177 100755
--- a/include/tensorflow/c/tf_status.h
+++ b/include/tensorflow/c/tf_status.h
@@ -17,7 +17,7 @@ limitations under the License.
 #define TENSORFLOW_C_TF_STATUS_H_
 
 #include "tensorflow/c/c_api_macros.h"
-#include "tsl/c/tsl_status.h"
+#include "external/local_tsl/tsl/c/tsl_status.h"
 
 #ifdef __cplusplus
 extern "C" {
diff --git a/include/tensorflow/core/platform/ctstring.h b/include/tensorflow/core/platform/ctstring.h
index 3b9359d..dafd79a 100755
--- a/include/tensorflow/core/platform/ctstring.h
+++ b/include/tensorflow/core/platform/ctstring.h
@@ -16,6 +16,6 @@ limitations under the License.
 #ifndef TENSORFLOW_CORE_PLATFORM_CTSTRING_H_
 #define TENSORFLOW_CORE_PLATFORM_CTSTRING_H_
 
-#include "tsl/platform/ctstring.h"
+#include "external/local_tsl/tsl/platform/ctstring.h"
 
 #endif  // TENSORFLOW_CORE_PLATFORM_CTSTRING_H_
diff --git a/include/tensorflow/core/platform/ctstring_internal.h b/include/tensorflow/core/platform/ctstring_internal.h
index c087dbc..5f164de 100755
--- a/include/tensorflow/core/platform/ctstring_internal.h
+++ b/include/tensorflow/core/platform/ctstring_internal.h
@@ -16,6 +16,6 @@ limitations under the License.
 #ifndef TENSORFLOW_CORE_PLATFORM_CTSTRING_INTERNAL_H_
 #define TENSORFLOW_CORE_PLATFORM_CTSTRING_INTERNAL_H_
 
-#include "tsl/platform/ctstring_internal.h"
+#include "external/local_tsl/tsl/platform/ctstring_internal.h"
 
 #endif  // TENSORFLOW_CORE_PLATFORM_CTSTRING_INTERNAL_H_
