diff --git a/tensorflow/tsl/protobuf/error_codes.proto b/tensorflow/tsl/protobuf/error_codes.proto
index c873d55..c77b281 100644
--- a/tensorflow/tsl/protobuf/error_codes.proto
+++ b/tensorflow/tsl/protobuf/error_codes.proto
@@ -9,7 +9,7 @@ option cc_enable_arenas = true;
 option java_outer_classname = "ErrorCodesProtos";
 option java_multiple_files = true;
 option java_package = "org.tensorflow.framework";
-option go_package = "github.com/google/tsl/tsl/go/protobuf/for_core_protos_go_proto";
+option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/tsl/protobuf/for_core_protos_go_proto";
 
 // The canonical error codes for TensorFlow APIs.
 //
