diff --git a/tensorflow/core/framework/dataset.proto b/tensorflow/core/framework/dataset.proto
index 9dfd03b..2ccc425 100644
--- a/tensorflow/core/framework/dataset.proto
+++ b/tensorflow/core/framework/dataset.proto
@@ -6,6 +6,8 @@ import "tensorflow/core/framework/tensor.proto";
 import "tensorflow/core/framework/tensor_shape.proto";
 import "tensorflow/core/framework/types.proto";
 
+option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/framework/dataset_go_proto";
+
 option cc_enable_arenas = true;
 
 // This file contains protocol buffers for working with tf.data Datasets.
diff --git a/tensorflow/core/framework/graph_debug_info.proto b/tensorflow/core/framework/graph_debug_info.proto
index 7af5262..e56f2e2 100644
--- a/tensorflow/core/framework/graph_debug_info.proto
+++ b/tensorflow/core/framework/graph_debug_info.proto
@@ -6,7 +6,7 @@ option cc_enable_arenas = true;
 option java_outer_classname = "GraphDebugInfoProtos";
 option java_multiple_files = true;
 option java_package = "org.tensorflow.framework";
-option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/protobuf/for_core_protos_go_proto";
+option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/framework/graph_debug_info_proto";

 message GraphDebugInfo {
   // This represents a file/line location in the source code.
diff --git a/tensorflow/core/framework/optimized_function_graph.proto b/tensorflow/core/framework/optimized_function_graph.proto
index d3a1c47..b08d845 100644
--- a/tensorflow/core/framework/optimized_function_graph.proto
+++ b/tensorflow/core/framework/optimized_function_graph.proto
@@ -5,6 +5,8 @@ package tensorflow;
 import "tensorflow/core/framework/graph.proto";
 import "tensorflow/core/framework/types.proto";
 
+option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/framework/optimized_function_graph_go_proto";
+
 // Optimized function graph after instantiation-related graph optimization
 // passes (up till before graph partitioning). The first half of the proto is
 // representing a GraphDef and the rest of the fields are extra information from
diff --git a/tensorflow/core/grappler/costs/op_performance_data.proto b/tensorflow/core/grappler/costs/op_performance_data.proto
index 5ef5fd9..c3e3ed7 100644
--- a/tensorflow/core/grappler/costs/op_performance_data.proto
+++ b/tensorflow/core/grappler/costs/op_performance_data.proto
@@ -17,6 +17,7 @@ syntax = "proto3";
 
 package tensorflow;
 option cc_enable_arenas = true;
+option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/grappler/costs/op_performance_data_proto_go_proto";
 
 import "tensorflow/core/framework/tensor.proto";
 import "tensorflow/core/framework/tensor_shape.proto";
diff --git a/tensorflow/core/lib/core/error_codes.proto b/tensorflow/core/lib/core/error_codes.proto
index d2b7e81..87e93d5 100644
--- a/tensorflow/core/lib/core/error_codes.proto
+++ b/tensorflow/core/lib/core/error_codes.proto
@@ -1,3 +1,5 @@
 syntax = "proto3";
 
 import public "tensorflow/tsl/protobuf/error_codes.proto";
+
+option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/lib/core/error_codes_proto_go_proto";
diff --git a/tensorflow/core/profiler/profiler_options.proto b/tensorflow/core/profiler/profiler_options.proto
index 8b0012f..01f8a2b 100644
--- a/tensorflow/core/profiler/profiler_options.proto
+++ b/tensorflow/core/profiler/profiler_options.proto
@@ -3,3 +3,5 @@ syntax = "proto3";
 package tensorflow.dummy;
 
 import public "tensorflow/tsl/profiler/protobuf/profiler_options.proto";
+
+option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/profiler/profiler_options_proto_go_proto";
diff --git a/tensorflow/core/profiler/protobuf/xplane.proto b/tensorflow/core/profiler/protobuf/xplane.proto
index 0dd575d..814eec4 100644
--- a/tensorflow/core/profiler/protobuf/xplane.proto
+++ b/tensorflow/core/profiler/protobuf/xplane.proto
@@ -3,3 +3,5 @@ syntax = "proto3";
 package tensorflow.profiler.empty;
 
 import public "tensorflow/tsl/profiler/protobuf/xplane.proto";
+
+option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/profiler/protobuf/for_profiler_protos_go_proto";
diff --git a/tensorflow/core/util/memmapped_file_system.proto b/tensorflow/core/util/memmapped_file_system.proto
index a988b45..587d8aa 100644
--- a/tensorflow/core/util/memmapped_file_system.proto
+++ b/tensorflow/core/util/memmapped_file_system.proto
@@ -17,6 +17,7 @@ syntax = "proto3";
 package tensorflow;
 
 option cc_enable_arenas = true;
+option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/util/memmapped_file_system_proto_go_proto";
 
 // A message that describes one region of memmapped file.
 message MemmappedFileSystemDirectoryElement {
diff --git a/tensorflow/core/util/saved_tensor_slice.proto b/tensorflow/core/util/saved_tensor_slice.proto
index 6278685..671ce1e 100644
--- a/tensorflow/core/util/saved_tensor_slice.proto
+++ b/tensorflow/core/util/saved_tensor_slice.proto
@@ -18,7 +18,9 @@
 syntax = "proto3";
 
 package tensorflow;
+
 option cc_enable_arenas = true;
+option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/util/saved_tensor_slice_proto_go_proto";
 option java_outer_classname = "SavedTensorSliceProtos";
 option java_multiple_files = true;
 option java_package = "org.tensorflow.util";
diff --git a/tensorflow/core/util/test_log.proto b/tensorflow/core/util/test_log.proto
index 20aa73a..203a41e 100644
--- a/tensorflow/core/util/test_log.proto
+++ b/tensorflow/core/util/test_log.proto
@@ -4,3 +4,5 @@ syntax = "proto3";
 package tensorflow.dummy;
 
 import public "tensorflow/tsl/protobuf/test_log.proto";
+
+option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/util/test_log_proto_go_proto";
